roles/engineblock/tasks/main.yml

```diff
@@ -66,6 +66,14 @@
       chdir: "{{ openconext_releases_dir }}/OpenConext-engineblock"
   changed_when: False
 
+- name: Create EngineBlock test database
+  mysql_db: name={{ engine_test_database_name }} state=present
+  tags: mysql
+
+- name: Create EngineBlock test database user
+  mysql_user: name={{ engine_test_database_user }} host={{ engine_test_database_host }} password={{ engine_test_database_password }} priv={{ engine_test_database_name }}.*:ALL state=present
+  tags: mysql
```

Comment:
Please add a "when: develop" here as well

Reply:
Ah yes, missed that one. Thanks for the feedback. I've replaced the mysql tag with the when: develop statement.